### PR TITLE
[ENH] Removes nested dataframes from shape dtw

### PR DESCRIPTION
**Pull request by TonyBagnall**
This removes the nested_univ from ShapeDTW (also fixes a couple of bugs I found there with "compound" option) and also removes doc string mentions to using nested Pandas dataframes. There is no usage of the nested_univ input type in classification, regression or clustering. There are still some transformations to do, and some testing to revise

#### Reference Issues/PRs
Part of #110

### CODE REVIEW

**Comment by MatthewMiddlehurst** [comment]
Test changes in setup.cfg got pushed 🙂 

Are there results from the previous version to compare against for ShapeDTW in the expected results file?

**Review by chrisholder — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -12,14 +12,6 @@ addopts =
     --ignore examples
     --ignore docs
     --doctest-modules
-    --durations 10
-    --timeout 600
-    --cov aeon
```
Comment (chrisholder) [inline setup.cfg:17]: you need to put these lines back in

Comment (TonyBagnall) [reply setup.cfg:17]: how many times have I done that! We should try fix it :)

**Comment by TonyBagnall** [comment]
> Test changes in setup.cfg got pushed 🙂
> 
> Are there results from the previous version to compare against for ShapeDTW in the expected results file?

dont think so, couldnt find any

**Comment by MatthewMiddlehurst** [comment]
looks like there is. results are in tests/_expected_outputs.py

image

**Review by MatthewMiddlehurst — APPROVED** [review]

**Pull request merged**